hazelcast/src/main/java/com/hazelcast/spi/Operations.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.spi;
+
+import com.hazelcast.cluster.impl.operations.JoinOperation;
+import com.hazelcast.cluster.impl.operations.WanReplicationOperation;
+import com.hazelcast.partition.MigrationCycleOperation;
+
+/**
+ * Utility class that contains helper methods related to {@link Operation}
+ */
+public final class Operations {
```

Comment:
Can you move this whole class into the com.hazelcast.spi.impl.operations package? Afaik user doesn't need to see this and we need to prevent exposing new API's if not needed